spec/factories/audit.rb

```diff
@@ -38,7 +38,7 @@
       audit.auditable_type = 'ApplicationWorkHistoryBreak'
       audit.auditable_id = evaluator.application_work_history_break.id
       audit.associated = evaluator.application_choice
-      audit.user_type = evaluator.user.class.to_s
+      audit.user_type = evaluator.user.class.to_s unless evaluator.username == '(Automated process)'
```

Comment:
So is user_type just nil if it's an Automated process? (just a question, trying to understand the audit model better, the PR look good to me)

Reply:
There's a strange thing happening here. If username is set to `(Automated process)` in the spec, it sets the user_type to `SupportUser` which I don't want for my spec and can't really make it nil because this after build is called 🙃 

This is more of a hack that hopefully won't bite us in the future

Reply:
Oh wow, ok, that is strange. Thanks for explaining it. I feel you are now that Audit expert.